src/python/Utils/PythonVersion.py

```diff
@@ -0,0 +1,8 @@
+"""
+Easily get the version of the python interpreter at runtime
+"""
+
```

Comment:
Python compatibility report requests
```
from __future__ import division
```
to be added. We might discuss the usefulness of this tomorrow btw

Reply:
Yeah, we can discuss this. I would remove such check, I think that `pylint --py3k` already warns us when we have a division in py2

Reply:
It should be W1619 at https://pylint.pycqa.org/en/latest/technical_reference/features.html

Reply:
For example, if we remove `from __future__ import division` from `SiteInfo.py` we get

```plainmtext
[dmwm@02f2a819e6dc WMCore]$ pylint --py3k src/python/WMCore/MicroService/Unified/SiteInfo.py
No config file found, using default configuration
************* Module src.python.WMCore.MicroService.Unified.SiteInfo
W: 12, 0: import missing `from __future__ import absolute_import` (no-absolute-import)
W: 52,37: division w/o __future__ statement (old-division)
W:357,35: division w/o __future__ statement (old-division)

------------------------------------------------------------------
Your code has been rated at 9.91/10 (previous run: 9.97/10, -0.06)
```

While if we remove it from `PerformanceMonitor.py` we get

```plaintext
[dmwm@02f2a819e6dc WMCore]$ pylint --py3k src/python/WMCore/WMRuntime/Monitors/PerformanceMonitor.py
No config file found, using default configuration
************* Module src.python.WMCore.WMRuntime.Monitors.PerformanceMonitor
W: 12, 0: import missing `from __future__ import absolute_import` (no-absolute-import)

-----------------------------------
Your code has been rated at 9.94/10
```

Because `PerformanceMonitor.py`casts the first argument to `float` and does not create problems with the old dfivision

Reply:
Thanks for this detailed information, Dario. Here is a new issue to get it removed in the coming weeks:
https://github.com/dmwm/WMCore/issues/10519